### ISSUE TITLE
Poll Devices on Startup?

### ISSUE THREAD

**Issue by erenfro** [MichaelB2018/smartthings-mqtt-bridge#4]
I kind of think that this should somehow poll all the devices on startup, especially if retain is on, and there's no retained topics for some devices, to help populate and build the basic construct of the mqtt system.

In example:
Fire Alarms. Not something you can generally trigger on demand like a simple contact door sensor. So, getting that to initially set a retained topic would require actually triggering the device's sensors with a smoke test and possibly carbon monoxide test, etc, just to get them to initially show up. Battery sensors as well, temperature sensors.

This is more especially important when handling this with HASS Discovery..

**Comment by MichaelB2018** [comment]
Agreed - issue is that I have so far only one type of device I can test this on (a blind). So if you have fire alarms or other devices, please feel free to send me a PR. Would be glad to include.
